src/ZaxJsonParser.h

```diff
@@ -17,11 +17,6 @@
 #ifndef _ZAXJSONPARSER_H_
 #define _ZAXJSONPARSER_H_
 
-#ifndef PRId64

```

Comment:
The new version still produces warnings with other versions of GCC compiler, so let's leave this as a define, like:
#ifndef PRId64
    #define PRId64 "lld"
#endif
This way we can change it manually for our needs easily, and as soon as a better picture is forming, we can have a more generic solution here.